compiler/modules/front/codeGen.bal

```diff
@@ -802,11 +808,14 @@ function codeGenAssignToVar(CodeGenContext cx, bir:BasicBlock startBlock, Enviro
         assignments = [ unnarrowedReg.number ];
     }
     s:Expr foldedExpr = check cx.foldExpr(env, expr, unnarrowedReg.semType);
-    var { result: operand, block: nextBlock } = check codeGenExpr(cx, startBlock, env, foldedExpr);
+    return { block: check genAssignInsn(cx, env, unnarrowedReg, startBlock, foldedExpr), assignments };
+}
 
-    bir:AssignInsn assign = { result: unnarrowedReg, operand };
-    nextBlock.insns.push(assign);
-    return { block: nextBlock, assignments };
+function genAssignInsn(CodeGenContext cx, Environment env, bir:Register result, bir:BasicBlock block, s:Expr expr) returns CodeGenError|bir:BasicBlock {
```

Comment:
This name is not consistent with the naming conventions in the file.  I suggest codeGenAssign

Reply:
Renamed